### PR TITLE
Bunch more info stuff

### PR DESCRIPTION
**Pull request by TheKevJames**

### CODE REVIEW

File: api/models/card.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                    {'iid': 1})
+
+    @requires_auth
+    def post(self):
+        """{"json": {"data": "(str, required)"}}"""
+        req = flask_restful.reqparse.RequestParser()
+        req.add_argument('data', type=str, card='json', required=True)
+        args = req.parse_args()
+
+        args['iid'] = 1
```
Comment (kyl191) [inline api/models/card.py:24]: We're fixing it to be 1?

File: api/models/contact.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    @requires_auth
+    def post(self):
+        """{"json": {"name": "(str, required)",
+                     "phone": "(str, required)"}}"""
+        req = flask_restful.reqparse.RequestParser()
+        req.add_argument('name', type=str, location='json', required=True)
+        req.add_argument('phone', type=str, location='json', required=True)
+        args = req.parse_args()
+
+        args['iid'] = 1
```
Comment (kyl191) [inline api/models/contact.py:26]: Again with the 1

**Review by kyl191 — APPROVED** [review]
Other than the fixed magic number 1, LGTM

**Comment by TheKevJames** [comment]
@kyl191 it's fixed for the same reason as the `/info` stuff is fixed. When/if we change that, we change this. Until then... fuck it.

**Pull request merged**